tools/lib/test_script.py

```diff
@@ -88,6 +89,18 @@ def assert_provisioning_status_ok(force: bool) -> None:
             sys.exit(1)
 
 
+def add_provision_check_override_param(parser: ArgumentParser, help: Optional[str] = None) -> None:
+    """
+    Registers --force argument to be used with various commands/tests in our tools.
+    """
+    if help is not None:
+        parser.add_argument("--force", action="store_true", help=help)
+    else:
+        parser.add_argument(
+            "--force", action="store_true", help="Run tests despite possible problems."
+        )
```

Comment:
Do we need this `help` parameter?  It doesn't seem that we ever pass it.

Reply:
Oh, I see, you pass it in `run-dev.py` -- I guess we can see if we can find phrasing that doesn't say "tests" to avoid needing this.